libvmaf/include/libvmaf/model.config.h

```diff
@@ -0,0 +1,19 @@
+#ifndef __VMAF_MODEL_CONFIG_H__
+#define __VMAF_MODEL_CONFIG_H__
+
+enum VmafModelFlags {
+    VMAF_MODEL_FLAG_DISABLE_CLIP = (1 << 0),
+    VMAF_MODEL_FLAG_ENABLE_TRANSFORM = (1 << 1),
+    VMAF_MODEL_FLAG_ENABLE_CONFIDENCE_INTERVAL = (1 << 2),
+    VMAF_MODEL_FLAG_DEFAULT = VMAF_MODEL_FLAG_DISABLE_CLIP,
```

Comment:
Call this `VMAF_MODEL_FLAGS_DEFAULT` since we may OR other flags later.

Reply:
Also, just put this in `model.h`, no need for the extra header file.

Reply:
Also, make `VMAF_MODEL_FLAGS_DEFAULT ` a `#define`, since it doesn't make sense for it to be part of the `enum`.